src/nunit.analyzers/Extensions/AttributeArgumentSyntaxExtensions.cs

```diff
@@ -179,33 +178,41 @@ private static bool HasBuiltInImplicitConversion(ITypeSymbol argumentType, IType
 
         private static bool CanBeTranslatedByTypeConverter(
             ITypeSymbol targetTypeSymbol,
-            object argumentValue)
+            object? argumentValue)
         {
             var typeName = targetTypeSymbol.GetFullMetadataName();
             var targetType = IntrinsicTypeConverters.FirstOrDefault(t => t.type.FullName == typeName);
 
-            if (targetType == default)
+            if (targetType != default)
             {
-                return false;
-            }
+                if (argumentValue is object)
+                {
+                    if (targetType.typeConverter == null)
+                    {
+                        return argumentValue.GetType() == typeof(string);
```

Comment:
(Unchanged code) Since `string` is a sealed type, this is the same as `argumentValue is string`.